docs/standard/linq/functional-programming-vs-imperative-programming.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
let's rename this as I proposed in the previous PR:
functional-vs-imperative-programming.md